lib/ClangImporter/ClangAdapter.cpp

```diff
@@ -563,6 +563,9 @@ OptionalTypeKind importer::translateNullability(clang::NullabilityKind kind) {
   case clang::NullabilityKind::Unspecified:
     return OptionalTypeKind::OTK_ImplicitlyUnwrappedOptional;
   }
+
+  // Work around MSVC warning: not all control paths return a value
+  llvm_unreachable("All switch cases are covered");
```

Comment:
We did this in Clang for a long time too to handle older GCCs that didn't have -Wswitch-enum. It'll be hard to get back in the habit but it doesn't seem like it needs to be MSVC-specific.